stewards.md

```diff
@@ -8,10 +8,12 @@ Stewards
 | Arnaud J Le Hors | [lehors][lehors] |
 | Tracy Kuhrt | [tkuhrt][tkuhrt] |
 | Vipin Bharathan | [vipinsun][vipinsun] |
+| Kamlesh Nagware | [knagware9][knagware9]
```

Comment:
```suggestion
| Kamlesh Nagware | [knagware9][knagware9] |
```

Reply:
I've been sorting these by GitHub username - please take Tracy's suggested edit, and move this to the top of the list